fluent.syntax/docs/api/ast.rst

```diff
@@ -0,0 +1,7 @@
+AST & Tooling
```

Comment:
Is there a way to exlcude some symbols from being documented? E.g. the `scalars_equal` function in `ast.py`.